src/main/java/com/rcv/Tabulator.java

```diff
@@ -818,6 +824,11 @@ private void recordSelectionForCastVoteRecord(
         continue;
       }
 
+      // check for a CVR with no rankings at all
+      if (cvr.rankToCandidateIDs.isEmpty()) {
```

Comment:
So all those people "voted" but didn't vote?

Reply:
Yeah. You can imagine this being pretty common when there are multiple contests on the ballot and some of them are more obscure.